File: SmartReceipts/ViewControllers/Notify/NotifyMasterSplitViewController.m
//
//  NotifyMasterSplitViewController.m
//  SmartReceipts
//
//  Created by Jaanus Siim on 16/05/15.
//  Copyright (c) 2015 Will Baumann. All rights reserved.
//

#import "NotifyMasterSplitViewController.h"

@interface NotifyMasterSplitViewController ()

@end

@implementation NotifyMasterSplitViewController

- (instancetype)initWithCoder:(NSCoder *)coder
{
    self = [super initWithCoder:coder];
    if (self) {
        self.preferredDisplayMode = UISplitViewControllerDisplayModeAllVisible;
    }
    return self;
}

- (void)viewWillAppear:(BOOL)animated {
    [super viewWillAppear:animated];
    [[self.viewControllers firstObject] viewWillAppear:animated];
}

@end

File: SmartReceipts/ViewControllers/Ad/NotifyMasterSplitViewController.swift
//
//  NotifyMasterSplitViewController.swift
//  SmartReceipts
//
//  Created by Bogdan Evsenev on 12/06/2017.
//  Copyright © 2017 Will Baumann. All rights reserved.
//


class NotifyMasterSplitViewController: UISplitViewController {
    
    required init?(coder aDecoder: NSCoder) {
        super.init(coder: aDecoder)
        preferredDisplayMode = .allVisible
    }
    
    override func viewWillAppear(_ animated: Bool) {
        super.viewWillAppear(animated)
        viewControllers.first?.viewWillAppear(animated)
    }
    
}
